centreon/www/front_src/src/ApiTokens/TokenCreation/Form.tsx

```diff
@@ -72,6 +74,16 @@ const FormCreation = ({
     values
   });
 
+  const platformFeatures = useAtomValue(platformFeaturesAtom);
+  const { canManageApiTokens, isAdmin } = useAtomValue(userAtom);
+
+  const getUsersEndpoint = (): string =>
+    platformFeatures?.isCloudPlatform && !isAdmin
+      ? getEndpointConfiguredUser({
+          search: { regex: { fields: ['is_admin'], value: '0' } }
```

Comment:
i can t get this well , is his not admin , we search an admin ?

Reply:
if we in the cloud platform & the user is not admin, we will search for all non-admin users